pep-0458.txt

```diff
@@ -1084,9 +1085,9 @@ information must be validated:
    since the last period should be discarded. As a result, developers of new
    projects will need to re-register their projects.
 
-3. If the packages themselves may have been tampered with, they can be
-   validated using the stored hash information for packages that existed at the
-   time of the last period.
+3. If the target files themselves may have been tampered with, they can be
```

Comment:
It feels disorientating to refer to target files and distributions in the same sentence when discussing the same thing

```suggestion
3. If the distribution files themselves may have been tampered with, they can be
```

Reply:
Makes sense. But I suggest to use "target files" twice then, because this sentence should also apply to non-distribution file target files, e.g. index files, right?